hydroflow_lang/src/graph/hydroflow_graph.rs

```diff
@@ -1519,6 +1531,9 @@ pub struct WriteConfig {
     /// Op text will only be their name instead of the whole source.
     #[cfg_attr(feature = "debugging", arg(long))]
     pub op_short_text: bool,
+    /// Op text will exclude any line that starts with "use".
+    #[cfg_attr(feature = "debugging", arg(long))]
+    pub op_text_no_imports: bool,
```

Comment:
Seems a bit overly specific? Whats this for?

Reply:
This was to remove the import statements that was included in each block in the mermaid graph. Is there a better way to do it?

Reply:
Does hydroflow+ generate a lot of `use`es?

Reply:
When I ran mermaid over the example in profiler.rs (commented out) I got a `use` for every node yeah